index.html

```diff
@@ -167,7 +167,39 @@ <h3 class="title">Skills</h3>
         </ul>
       </div>
     </section>
+
   </main>
+  <footer class="footer">
+    <div class="footer-shapes"></div>
+    <div class="footer-content">
+      <div class="contact-intro">
+        <div class="title">
+          Contact me
+        </div>
+        <div class="description">
+          If you have an application you are interested in developing, a feature that you need built or a project that
+          needs coding. I'd love to help with it
+        </div>
+      </div>
+      <form action="https://formspree.io/f/xbjwdrql" method="post" class="contact-form">
+        <ul>
+          <li>
+            <input type="text" placeholder="Enter your Name" maxlength="30" required />
+          </li>
+          <li>
+            <input type="email" placeholder="Enter your email" required />
+          </li>
+          <li>
+            <textarea name="message" id="contact-message" cols="30" rows="7" maxlength="500"
+              placeholder="Write your message here" required></textarea>
+          </li>
+          <li>
+            <button type="submit" class="submit-btn">Get in Touch</button>
+          </li>
+        </ul>
+      </form>
+    </div>
+  </footer>

```

Comment:
There is a white space on the right side of your about and form section that should not be there 🧐

![Captura](https://user-images.githubusercontent.com/15807343/149219410-34d9d4c3-1fa7-48ca-b0f4-0ebfd55c5a0d.PNG) | ![Captura1](https://user-images.githubusercontent.com/15807343/149219465-504fe5eb-2a2e-4d4f-b0ae-8a7fea2727ab.PNG)

Kindly fix the width for these sections to meet with the proper design 🙌